web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -1934,3 +1963,153 @@ form#views-exposed-form-job-board-page {
     margin-left: 0px;
   }
 }
+
+.gm-style .gm-style-iw-c {
+  background-color: rgba(42, 85, 107, 0.85);
+  border: 2px solid #FF9049;
+  border-radius: 0px;
+  padding-left: 20px;
+  padding-right: 20px !important;
+  padding-bottom: 12px !important;
+  box-shadow: none !important;
+
+  .directory-vol-role-count {
+    color: #FFFFFF;
+  }
+
+  .directory-like-count {
+    color: #FFFFFF;
+  }
+
+  .directory-paid-role-count {
+    padding-left: 0;
+    color: #FFFFFF;
+  }
+
+  @media (max-width: $md) {
+    .field-name-employer-description-summary {
+      display: none;
+    }
+  }
+}
+
+.gm-style {
+  .gm-style-iw {
+    font-family: Lato;
+    font-weight: 300;
+    font-size: 13px;
+
+    h2, h2 a {
+      font-size: 14px;
+      margin: 8px 0 6px 0;
+      color: #FFFFFF;
+      text-transform: uppercase;
+
+      &:hover {
+        color: #569BDD;
+      }
+    }
+  }
+}
+
+.gm-style .gm-style-iw h2, .h2 {
```

Comment:
I think you mean 

```scss
.gm-style .gm-style-iw h2,  .gm-style .gm-style-iw  .h2 {
```
What you've written will apply to the .h2 selector everywhere - not just in the popup. This does also fit inside the block above it.